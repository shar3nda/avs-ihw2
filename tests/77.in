QS*v\@Jd^ev9c'(,G|YThHN$6 4@8'SwY]
fkf	k6!0/J9L11"jULjb2
9~FHMhrGn#TP*'UOC[1 _