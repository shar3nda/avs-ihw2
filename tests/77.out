Lowercase: 986, uppercase: 953